organizationdomain/client.go

```diff
@@ -0,0 +1,122 @@
+// Package organizationdomain provides the Organization Domains API.
+package organizationdomain
+
+import (
+	"context"
+	"net/http"
+	"net/url"
+	"strconv"
+
+	"github.com/clerk/clerk-sdk-go/v2"
+)
+
+//go:generate go run ../cmd/gen/main.go
+
+const path = "/organizations"
+
+// Client is used to invoke the Organization Domains API.
+type Client struct {
+	Backend clerk.Backend
+}
+
+func NewClient(config *clerk.ClientConfig) *Client {
+	return &Client{
+		Backend: clerk.NewBackend(&config.BackendConfig),
+	}
+}
+
+type CreateParams struct {
```

Comment:
:wrench: All fields should be pointers and omitted when empty. This is the convention used throughout the library. See the organization package for an example: https://github.com/clerk/clerk-sdk-go/blob/807c12461675e18bb88c5162b99cd3951e803902/organization/client.go#L32